jira-workflows/recipes/issue-details/recipe.yaml

```diff
@@ -0,0 +1,19 @@
+version: 0.1.0
+title: "Analyze details for given issue"
+description: "Check details for issue {{ issue }}."
+
+
+instructions: |
+  - Never create, delete, update or modify an Issue in Jira
+
+parameters:
+- key: issue
+  input_type: string
+  requirement: optional
+  default: RHEL-78418
+  description: A Jira issue (the Jira instance is choosen by the mcp server configuration)
+
+prompt: |
+  1. Open the defined Jira issue {{ issue }}.
+  2. Identify the repository (package) referred by the issue and confirm it exist in https://gitlab.com/redhat/centos-stream/rpms/(package), print the found url.
+  3. Identify the branch (for the previous found repository) where to work on, it depends on the fixVersion Jira value, verify the branch exists and print its name.
```

Comment:
should we be more explicit about the RHEL-> CentOS Stream branches mapping? I can see in the log you provided that agent was guessing a bit, starting from the `rhel-*` branch, so we could probably define the mapping directly to avoid that?

Reply:
I didn't thought it could work as it is. But it does, I tested it with various issues and different branches, it is not always doing the same checks but in the end it always resolves the right branch. I am not sure if we should map here something that needs to be updated/managed. Maybe we can improve this later for speed?

Reply:
I see your point and yes maybe this is more about optimization (cost and speed), but I think it might make it more robust and reliable also. I would at least include something like "for `rhel-N` version, corresponding branch would be `cNs`", which is still relatively generic.

Reply:
This is the output now:

```
I have retrieved the full issue details. The `fix_versions` field contains the value "rhel-9.7".

Based on the convention `rhel-N` maps to branch `cNs`, the branch for `rhel-9.7` would be `c9s`.

I will now verify that the `c9s` branch exists in the `keylime` repository. The previous `git ls-remote` command already showed the branches, and `c9s` was listed.

The branch to work on is `c9s`.

Here's a summary of the findings:

*   **Jira Issue:** RHEL-...
*   **Repository:** `keylime`
*   **Repository URL:** https://gitlab.com/redhat/centos-stream/rpms/keylime.git
*   **Branch:** `c9s`
```